pyproject.toml

```
@@ -2,3 +2,105 @@
 line-length = 130
 target-version = ['py37', 'py38', 'py39', 'py310']
 include = '(app|migrations|tests)/.*\.pyi?$'
+
+[tool.poetry]
+name = "notification-api"
+version = "0.1.0"
+description = "Public-facing REST API for Notification built on the GOV.UK Notify platform."
+authors = ["Canadian Digital Service"]
+license = "MIT license"
+readme = "README.md"
+packages = []
+
+[tool.poetry.scripts]
+notify-api = ""
+
+[build-system]
+requires = ["poetry>=1.3.2"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.dependencies]
+python = "^3.10.9"
```

We might want the tilde instead of the caret, for accepting patches and nothing more. The caret could upgrade from 3.10 to 3.11 [if read the doc correctly](https://python-poetry.org/docs/dependency-specification/).

```suggestion
python = "~3.10.9"
```

---

Agreed, good catch.

---

Fixed - as well in `admin` and `DDL-API`.